pkg/gen/input/workflows/internal/file/create_release_pr.go

```diff
@@ -28,7 +28,7 @@ on:
     branches:
       - 'master#release#v*.*.*'
       - 'legacy#release#v*.*.*'
-      - 'release-v*.x.x#release#v*.*.*'
+      - 'release-v*.*.x#release#v*.*.*'
```

Comment:
To be even more rigid:

```suggestion
      - 'release-v*.*.x#release#v*.*.*'
      # "!" negates previous positive patterns so it has to be at the end.
      - '!release-v*.x.x#release#v*.*.*'
```